infra/main.parameters.json

```diff
@@ -116,6 +116,9 @@
     "embeddingDimensions": {
       "value": "${AZURE_OPENAI_EMB_DIMENSIONS}"
     },
+    "gpt4vDeploymentCapacity":{
+      "value": "${AZURE_OPENAI_GPT4V_DEPLOYMENT_CAPACITY=10}"
```

Comment:
Also unrelated, I needed to be able to set capacity to get the vision endpoint deployed to our restrictive tenants. I can move to other branch if desired.